chainio/clients/avsregistry/reader.go

```diff
@@ -355,6 +355,26 @@ func (r *ChainReader) GetOperatorFromId(
 	return operatorAddress, nil
 }
 
+func (r *ChainReader) QueryRegistrationDetail(
+	opts *bind.CallOpts,
+	operatorAddress common.Address,
+) (map[int]bool, error) {
+	operatorId, err := r.GetOperatorId(opts, operatorAddress)
+	if err != nil {
+		return nil, err
+	}
+	value, err := r.registryCoordinator.GetCurrentQuorumBitmap(opts, operatorId)
+	if err != nil {
+		return nil, err
+	}
+	numBits := value.BitLen()
+	quorums := make(map[int]bool, numBits)
+	for i := 0; i < numBits; i++ {
+		quorums[i] = value.Int64()&(1<<i) != 0
```

Comment:
Haven’t used quorum bitmaps in a long while. Are you sure this is how they work? Aka the quorums are laid out from right to left and not the other way around?

Reply:
Yep, i have tested 2 cases:
1. opt-in into 3 quorums: 0,1,2 then eject operator from 1, the contract returns 5 whose bit format is: 1,0,1
2. opt-in into 3 quorums: 0,1,2 them eject operator from 0, the contract returns 6 whose bit format is: 1,1,0
<img width="1022" alt="image" src="https://github.com/user-attachments/assets/2b0aaa8b-ee35-4f82-8286-519b7b8c4f8f">
<img width="1066" alt="image" src="https://github.com/user-attachments/assets/beb31dc2-2354-4db2-b557-61291b9c3e26">

Reply:
Awesome will trust you on this one then